File: from-1501-to-1600/1550-three-consecutive-odds/solution.py
class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        odds = 0
        for num in arr:
            if num % 2:
                odds += 1
                if odds == 3:
                    return True
            else:
                odds = 0

        return False

File: from-1501-to-1600/1550-three-consecutive-odds/solution.go
func threeConsecutiveOdds(arr []int) bool {
    checkSum := 0
    for _, num := range arr {
        if num % 2 == 0 {
            checkSum = 0
        } else {
            checkSum += 1
            if checkSum == 3 {
                return true
            }
        }
    }
    return false
}